### DISCUSSION TITLE
How to re-lock bootloader and unroot phone?

### DISCUSSION THREAD

**Discussion by Derpitron** [driedpampas/realme-8-megaguide#28] [Q&A]
I have not tried your guide yet. I am not experienced with rooting/unlocking for realme/oppo phones.

But due to my banking apps and software updates I need to be able to return back to normal phone mode if needed. I am not concerned about data loss/wiping, since I back up everything already.

How can I reverse this process, and relock my bootloader/unroot the phone?

**Comment by driedpampas** [comment]
This is super super late but I just compiled the steps you'd need to take in [Reverting](https://github.com/driedpampas/realme-8-megaguide/wiki/Reverting)
